lib/anyway/rails/loaders/yaml.rb

```diff
@@ -5,11 +5,18 @@ module Rails
     module Loaders
       class YAML < Anyway::Loaders::YAML
         def load_base_yml(*)
+          parsed_yml = super
+          return parsed_yml unless environmental?(parsed_yml)
+
           super[::Rails.env] || {}
         end
 
         private
 
+        def environmental?(parsed_yml)
+          parsed_yml.keys.any? { |key| ["test", "development", "production", ::Rails.env].include?(key) }
```

Comment:
Maybe, we should move the list of known env to configuration? (somewhere in `anyway/rails/settings`).

That has two advantages:
- We can re-use the same array object instead of creating a new one every time
- We can make this list extendable and configurable (e.g., `config.anyway_config.known_environments << 'staging').